src/main/resources/styles/reporting.css

```diff
@@ -57,6 +57,10 @@
 	background-color: #C5D88A;
 }
 
+.hidden {
+	visibility: hidden;
```

Comment:
When a step will have the hidden attribute in Json set to true then the
container div will have the class set to true, thus overriding the status
passsed or failed with hidden, making that step to be hidden from the
report since the display of that step would be redundant, as you can note
from the screenshots I have attached.
On May 13, 2015 9:42 PM, "Damian Szczepanik" <notifications@github.com>
wrote:

> In src/main/resources/styles/reporting.css
> <https://github.com/masterthought/cucumber-reporting/pull/101#discussion_r30262309>
> :
>
> > @@ -57,6 +57,10 @@
> >  	background-color: #C5D88A;
> >  }
> >
> > +.hidden {
> > +	visibility: hidden;
>
> where this class is used?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/masterthought/cucumber-reporting/pull/101/files#r30262309>
> .
>